dls-controls/malcolmjs
# Feature/high freq inputs #262

## Description

Moved the web socket into the web worker to improve performance, this means all communications with the server now happens in a separate thread.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- run the site and go to a layout to see that it is more responsive, open the SEQ1 block and see that the updates happen quickly.

## Agile board tracking

connect to #262 
closes #262
